meteor/lib/collections/StudioInstallations.ts

```diff
@@ -34,8 +34,8 @@ export interface Mappings {
 export interface Mapping {
 	device: PlayoutDeviceType,
 	lookahead: LookaheadMode,
-	deviceId: string
-	// [key: string]: any
+	deviceId: string,
+	[key: string]: any
```

Comment:
@nytamin you had commented out this, I added it back. Can't see better ways of getting typings for all device types.

Reply:
I believe the reason I commented it out was to keep a strict typing. If you want to do a specific typing you should use the MappingLawo, MappingAtem etc, not the Mapping itself.
If you add a `[key: string]: any` you essentially disable type checks for properties in that interface, which is bad.

Reply:
I know, but I didn't make the effort. 
You still enforce the deviceID and other strict ones, so that part of the interface still holds.